app/models/manager_refresh/inventory_object.rb

```diff
@@ -81,45 +81,22 @@ def dependency?
       true
     end
 
-    def allowed_writers
-      return [] unless model_class
-
```

Comment:
TBH, if we keep the automatic way, people will not create the attribute lists in `app/models/manager_refresh/inventory/core.rb ` - and this is something I really want, also for documentation purposes